apps/developer-hub/src/chapters/ChapterHelper/DAOChapter.tsx

```diff
@@ -0,0 +1,286 @@
+import Deso from 'deso-protocol';
+import {
+  DAOCoinLimitOrderOperationTypeString,
+  DAOCoinLimitOrderWithCancelOrderIDRequest,
+  DAOCoinLimitOrderWithExchangeRateAndQuantityRequest,
+  DAOCoinRequest,
+  GetDAOCoinLimitOrdersRequest,
+  GetTransactorDAOCoinLimitOrdersRequest,
+  TransferDAOCoinRequest,
+} from 'deso-protocol-types';
+import { Route } from 'react-router-dom';
+import { ParentRoutes } from '../../services/utils';
+import Page from '../Read/Page';
+import { CHAPTERS } from './Chapter.models';
+import { PageSection } from './PageSections';
+const deso = new Deso();
+export const daoChapter = {
+  DAO_COIN: {
+    parentRoute: ParentRoutes.dao,
+    title: 'DAO Coin',
+    route: '/dao/dao-coin',
+    params: () => {
+      return {
+        UpdaterPublicKeyBase58Check: localStorage.getItem('login_key'),
+        ProfilePublicKeyBase58CheckOrUsername:
+          localStorage.getItem('login_key'),
+        OperationType: 'mint',
+        CoinsToMintNanos: '0x3B9ACA00',
+      } as Partial<DAOCoinRequest>;
+    },
+    method: deso.dao.DAOCoin,
+    documentation: [
+      'https://docs.deso.org/for-developers/backend/transactions/construct-transactions/dao-transactions-api#dao-coin',
+    ],
+    githubSource: [],
+    component: function () {
+      return (
+        <Route
+          key={this.title}
+          path={this.route}
+          element={
+            <Page
+              method={{
+                methodName: `deso.dao.DAOCoin(request)`,
+                params: this.params,
+                method: this.method,
+              }}
+              pretext={PageSection(
+                this.title,
+                <div>
+                  DAO Coin is used to mint, burn, update transfer restriction
+                  status, and disable minting
+                </div>
+              )}
+              demo={true}
+              chapters={CHAPTERS}
+              selectedChapter={this}
+              bind="dao"
+            />
+          }
+        ></Route>
+      );
+    },
+  },
+  TRANSFER_DAO_COIN: {
+    parentRoute: ParentRoutes.dao,
+    title: 'Transfer DAO Coin',
+    route: '/dao/transfer-dao-coin',
+    params: () => {
+      return {
+        SenderPublicKeyBase58Check: localStorage.getItem('login_key'),
```

Comment:
I know some places use local storage here, but lets switch it to deso.identity.getUserKey().

Reply:
Same for any reoccurrences in this file